client/src/Movies/Movie.js

```diff
@@ -9,7 +10,7 @@ export default class MovieCard extends React.Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id =  this.props.match.params.id;
```

Comment:
You can use object destructuring here `const { id } =  this.props.match.params;`